localgov_theme.theme

```diff
@@ -140,25 +140,28 @@ function localgov_theme_preprocess_office_hours(&$variables) {
  */
 function localgov_theme_theme_suggestions_alter(array &$suggestions, array $variables, $hook) {
   // Allow page template to be overridden by content type.
-  $node = \Drupal::routeMatch()->getParameter('node');
-  if (!$node instanceof NodeInterface) {
-    $node = Node::load((int) $node);
-  }
-  // For the frontpage, ensure that *page__front* is always the final
-  // suggestion.
-  $is_front_page = end($suggestions) === 'page__front';
-  if ($is_front_page) {
-    $front_page_tpl_suggestion_key = key($suggestions);
-    unset($suggestions[$front_page_tpl_suggestion_key]);
-
-    if ($node instanceof NodeInterface) {
+  if ($hook == 'page') {
```

Comment:
Hi Andy, I think this can still land us into trouble given that not all pages are node pages.  The original code was ```if ($hook == 'page' && $node = \Drupal::routeMatch()->getParameter('node')) {``` which may be what we need.  I am particularly worried about line 145-147 which assumes we are in a node page.

Reply:
@Adnan-cds I've added that nowm, let me if this still works for you and on the revision pages.

Reply:
This version works without any issues. Thanks a lot :)